ironfish-cli/src/commands/evm/deploy-contract.ts

```diff
@@ -32,26 +32,15 @@ export class TestEvmCommand extends IronfishCommand {
     await node.chain.blockchainDb.stateManager.putAccount(senderAddress, senderAccount)
     await node.chain.blockchainDb.stateManager.commit()
 
-    const globalContractAddress = Address.fromString(
-      '0xffffffffffffffffffffffffffffffffffffffff',
-    )
-
-    await node.chain.blockchainDb.stateManager.checkpoint()
-    await node.chain.blockchainDb.stateManager.putContractCode(
```

Comment:
should we remove this command altogether since it doesn't deploy a contract anymore? or leave it for now and reuse it for deploying a test ERC20?